WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/usecases/FetchScanStateUseCase.kt

```diff
@@ -27,32 +26,36 @@ class FetchScanStateUseCase @Inject constructor(
     suspend fun fetchScanState(
         site: SiteModel,
         polling: Boolean = true,
-        delayInMs: Long = FETCH_SCAN_STATE_DELAY_MILLIS
+        delayInMs: Long = FETCH_SCAN_STATE_DELAY_MILLIS,
+        startAfterDelay: Boolean = false
```

Comment:
Minor (🔍): This is totally opinionated so feel free to ignore. I would have named this like:
- `startWithDelay` or,
- `startDelay`, removing the `After` word altogether.

PS: I don't know why but `After` sounds a bit tricky, maybe I am used to see this kind of parameters with `With` instead. Again, please ignore if you feel `After` is much better.

Reply:
In 43afa89